src/drivers/uavcan_v1/Uavcan.cpp

```diff
@@ -214,13 +232,24 @@ void UavcanNode::Run()
 
 		// update parameters from storage
 		updateParams();
+
+		for (auto &publisher : _publishers) {
+			// Have the publisher update its associated port-id parameter
+			// Setting to 0 disable publication
```

Comment:
0 is a valid port-ID. The Spec [says that 0xFFFF should be used to represent unconfigured port-ID](https://github.com/UAVCAN/public_regulated_data_types/blob/342f3587d76ee48fd381c55929b7317942e481c3/uavcan/register/384.Access.1.0.uavcan#L89).

Reply:
Ok, simple to fix - what would be a good "invalid" default value to use here?

Reply:
65535

Reply:
Will fix